callstats-java-sdk/src/main/java/io/callstats/sdk/BridgeStatusInfo.java

```diff
@@ -29,32 +35,74 @@
 	/** The avg interval jitter. */
 	private float avgIntervalJitter;
 	
+	/** Download bit rate for the video bridge in kilobits per second.*/
+	private int downloadBitRate;
+	
+	/** Upload bit rate for the video bridge in kilobits per second.*/
+	private int uploadBitRate;
+	
+	/** The value is between 0 and 1 and represents the RTP packet loss for the video bridge. */
+	private float rtpLoss;
+	
+	/** Number of audio channels. */
+	private int audioChannelsCount;
+	
+	/** Number of video channels. */
+	private int videoChannelsCount;
+	
+	/** Number of video conferences. */
+	private int conferenceCount;
+	
+	/** Number of video participants. */
+	private int participantsCount;	
+	
 	/**
 	 * Instantiates a new bridge status info.
 	 *
 	 * @param cpuUsage the cpu usage
 	 * @param memoryUsage the memory usage
+	 * @param totalMemory the total memory
+	 * @param threadCount the thread count
 	 * @param sentBytes the sent bytes
 	 * @param receivedBytes the received bytes
 	 * @param intervalLoss the interval loss
 	 * @param totalLoss the total loss
 	 * @param avgIntervalRtt the avg interval rtt
 	 * @param avgIntervalJitter the avg interval jitter
+	 * @param downloadBitRate the download bit rate
+	 * @param uploadBitRate the upload bit rate
+	 * @param rtpLoss the rtp loss
+	 * @param audioChannelsCount the audio channels count
+	 * @param videoChannelsCount the video channels count
+	 * @param conferenceCount the conference count
+	 * @param participantsCount the participants count
 	 */
-	public BridgeStatusInfo(float cpuUsage, float memoryUsage, long sentBytes,
+	public BridgeStatusInfo(float cpuUsage, float memoryUsage,
```

Comment:
This constructor's parameter is too long. Besides, since there is a ```BridgeStatusInfoBuilder``` to create the ```BridgeStatusInfo``` object, we can even make the constructor private. See the following example for the builder pattern.

Ref: http://javarevisited.blogspot.gr/2012/06/builder-design-pattern-in-java-example.html